frontend/src/components/uam-configure-group/uam-configure-group.tsx

```diff
@@ -0,0 +1,147 @@
+import { AppRoute, PaginationDefaultValue } from 'common/enums/enums';
+import { FC, GroupsConfigureRequestDto } from 'common/types/types';
+import { Button, Input } from 'components/common/common';
+import {
+  useAppDispatch,
+  useAppForm,
+  useAppSelector,
+  useEffect,
+  usePagination,
+  useParams,
+  useSelectedItems,
+} from 'hooks/hooks';
+import { uamConfigureGroupActions } from 'store/actions';
+import { groupConfigureClient as groupConfigureClientSchema } from 'validation-schemas/validation-schemas';
+
+import { DEFAULT_CONFIGURE_GROUP_PAYLOAD } from './common/default-configure-group-payload';
+import { GroupConfigureFieldsName } from './common/enums/enums';
+import { PermissionsTable } from './components/permissions-table/permissions-table';
+import { UsersTable } from './components/users-table/users-table';
+import styles from './styles.module.scss';
+
+const UAMConfigureGroup: FC = () => {
+  const dispatch = useAppDispatch();
+  const { id } = useParams();
+  const isEdit = Boolean(id);
+  const { permissions, group, users } = useAppSelector(
+    (state) => state.uamConfigureGroup,
+  );
+  const { control, handleSubmit, errors, reset } =
+    useAppForm<GroupsConfigureRequestDto>({
+      defaultValues: DEFAULT_CONFIGURE_GROUP_PAYLOAD,
+      validationSchema: groupConfigureClientSchema,
+    });
+  const {
+    items: permissionIds,
+    handleToggle: handlePermissionToggle,
+    setItems: setDefaultPermissionIds,
+  } = useSelectedItems<number>(group?.permissionIds ?? []);
+  const {
+    items: userIds,
+    handleToggle: handleUserToggle,
+    setItems: setDefaultUserIds,
+  } = useSelectedItems<number>(group?.userIds ?? []);
+  const { page: usersPage, handlePageChange: handleUsersPageChange } =
+    usePagination({ queryName: 'users' });
+
+  const onSubmit = (): void => {
```

Comment:
```suggestion
  const handleSubmit = (): void => {
```
`handle` for local component's cbs